src/applications/static-pages/health-care-manage-benefits/schedule-view-va-appointments-page/components/AuthContent/index.js

```diff
@@ -38,7 +54,7 @@ export const AuthContent = () => (
         View, schedule, or cancel a VA appointment&nbsp;online
       </h2>
     </div>
-    <CallToActionWidget appId="view-appointments" setFocus={false} />
+    <CernerCallToAction callToActions={callToActions} type="these" />
```

Comment:
This made me lol a bit but it's just due to the design spec content which has a high probability of changing before going to prod